src/components/sdk/PreviewVideo/index.tsx

```diff
@@ -64,7 +64,7 @@ export const PreviewVideo: React.FC<BaseSdkProps> = (props) => {
       }
     }
     startPreview();
-  }, [device]);
+  }, [meetingManager, device, audioVideo, setIsVideoEnabled]);
```

Comment:
I think we may want to combine these two useEffects - I shouldn't have separated them before.

Reply:
Actually I think I'm fine with this. Since that useEffect comes first, if the audioVideo is ever changed, we'd first stop the VideoPreview, `setIsVideoEnabled` to false, and then `selectVideoInput`, then `startVideoPreview`, then `setIsVideoEnabled` = true